pandas/core/indexes/base.py

```diff
@@ -5935,3 +5941,22 @@ def _maybe_asobject(dtype, klass, data, copy: bool, name: Label, **kwargs):
         return index.astype(object)
 
     return klass(data, dtype=dtype, copy=copy, name=name, **kwargs)
+
+
+def get_unanimous_names(*indexes: Type[Index]) -> Tuple[Any, ...]:
```

Comment:
`Type[Index]` seems really weird here.  Shouldnt it be `Tuple[Index]`?  And i think the return type should use Label instead of Any

Reply:
Ah, I was wondering if there was a `Label` notion! And now I see there's a `_typing.py`, will keep that handy in the future.

Yeah, I don't know why that extraneous ended up `Type` there. But actually with an *args situation, you just give the type of the individual elements, so I think it should just be `Index`.